setup_new_function.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env python
```

Comment:
This file could use a comment at the top explaining what it's useful for, how to use it, and what to do if it stops working with future releases of boto, botocore, etc., (e.g. "Please fix this if it breaks under future versions of boto libraries").

I'd also recommend moving this into the ./scripts/ directory and adding an entry to `Makefile` to invoke it. Something like:
```
scaffold:
  pip install -r requirements-dev.txt
  python scripts/setup_new_function.py
```

Maybe it's better named as ./scripts/scaffold.py or ./scripts/generate_scaffold.py? Your call.

Reply:
got it!
I'll do that :)

Reply:
I also move `implementation_coverage.py` to `scripts` directory.